src/shared/src/components/AltinnTextArea.test.tsx

```diff
@@ -14,13 +14,13 @@ const render = (props = {}) => {
 };
```

Comment:
denne er sletta, da komponenten ikke var i bruk...

Reply:
Yeah, saw that. Merged now :)